ProjectTodo/src/containers/CardScreen.js

```diff
@@ -40,9 +65,15 @@ export default class CardScreen extends React.Component {
           checked={this.state.done}
           onPress={() => this.setState({ done: !this.state.done })}
         />
-        <Button title="Edit" onPress={this.editCard} />
-        <Button title="Delete" onPress={this.deleteCard} />
 
+        {this.state.editForm ? (
+          <Button title="Edit" onPress={this.editCard} />
+        ) : (
+          <Button title="Create" onPress={this.newCard} />
+        )}
+        {this.state.editForm ? (
+          <Button title="Delete" onPress={this.deleteCard} />
+        ) : null}
```

Comment:
mozes this.state.editing form && <Button> onda ne moras kao ternarni da stavljas : null